main/test/ca/uwaterloo/flix/language/phase/TestParser.scala

```diff
@@ -1052,6 +1052,24 @@ class TestParser extends FunSuite with TestUtils {
     expectError[NonExhaustiveMatchError](new Flix().addStr(input).compile())
   }
 
+  test("Expression.Spawn.01") {
```

Comment:
I would just remove these three errors. We don't really have to test that there is some specific input that we cannot parse. 

Negative tests are mostly relevant for later stages of the compiler where you want to be sure that some program is definitely rejected, e.g. by the typer.